### PR TITLE
[zh-cn]: update the translation of `currentTarget`

### PR DESCRIPTION
**Pull request by tianyeeT**

### CODE REVIEW

File: files/zh-cn/web/api/event/currenttarget/index.md [hunk 1]
```diff
@@ -1,44 +1,82 @@
...
+
+const reset = document.querySelector("#reset");
+reset.addEventListener("click", () => document.location.reload());
+```
+
+#### 结果
+
+如果你点击子 `<div>` 中，则 `target` 将标识子元素。如果你点击父 `<div>` 中，则 `target` 将标识父元素。
+
+在这两种情况下，`currentTarget` 都标识父元素，因为那是处理程序附加到的元素。
```
Comment (yin1999) [inline files/zh-cn/web/api/event/currenttarget/index.md:77]:
```suggestion
在这两种情况下，`currentTarget` 都标识父元素，因为那是处理器附加到的元素。
```

File: files/zh-cn/web/api/event/currenttarget/index.md [hunk 2]
```diff
@@ -48,6 +86,7 @@ document.body.addEventListener("click", hide, false);
 
 {{Compat}}
 
-## 参考
+## 参见
 
-- [Comparison of Event Targets](/zh-CN/docs/Web/API/Event/Comparison_of_Event_Targets)
+- [事件目标的比较](/zh-CN/docs/Web/API/Event/Comparison_of_Event_Targets)
+- [事件冒泡](/zh-CN/docs/Learn/JavaScript/Building_blocks/Events#event_bubbling)
```
Comment (yin1999) [inline files/zh-cn/web/api/event/currenttarget/index.md:92]:
```suggestion
- [事件冒泡](/zh-CN/docs/Learn/JavaScript/Building_blocks/Events#事件冒泡)
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/api/event/currenttarget/index.md [hunk 3]
```diff
@@ -1,44 +1,82 @@
...
-{{domxref("Event")}} 接口的只读属性 **`currentTarget`** 表示的，标识是当事件沿着 DOM 触发时事件的当前目标。它总是指向事件绑定的元素，而 {{domxref("Event.target")}} 则是事件触发的元素。
+**`currentTarget`** 是 {{domxref("Event")}} 接口的一个只读属性，用于标识事件处理程序所附加的元素。
 
-## 语法
+这并不总是与触发事件的元素相同，因为事件可能在具有处理程序的元素的后代上触发，然后通过事件[冒泡](/zh-CN/docs/Learn/JavaScript/Building_blocks/Events#event_bubbling)到具有处理程序的元素。事件的触发元素由 {{domxref("Event.target")}} 给出。
 
-```plain
-var currentEventTarget = event.currentTarget;
-```
+请注意，`currentTarget` 的值仅在事件处理程序中可用。在事件处理程序外部，它将为 `null`。这意味着，例如，如果在事件处理程序内部引用 `Event` 对象然后在事件处理程序外部访问其 `currentTarget` 属性，则其值将为 `null`。
```
[the remark this answers is not in the document]
Comment (tianyeeT) [reply files/zh-cn/web/api/event/currenttarget/index.md:14]: https://github.com/mdn/content/pull/33798

**Comment by tianyeeT** [comment]
https://github.com/mdn/content/pull/33798
need to follow up

**Comment by yin1999** [comment]
@tianyeeT 麻烦更新一下文档

File: files/zh-cn/web/api/event/currenttarget/index.md [hunk 4]
```diff
@@ -1,44 +1,82 @@
 ---
-title: Event.currentTarget
+title: Event：currentTarget 属性
```
Comment (tianyeeT) [inline files/zh-cn/web/api/event/currenttarget/index.md:2]:
```suggestion
title: Event.currentTarget
```
@yin1999 要改回去吗

Comment (yin1999) [reply files/zh-cn/web/api/event/currenttarget/index.md:2]: 与英文文档保持一致，不用改

**Review by yin1999 — APPROVED** [review]

**Pull request merged**